clang/test/Driver/sycl-bfloat16-lib-win.cpp

```diff
@@ -0,0 +1,62 @@
+///
+/// Check if bfloat16 native and fallback libraries are added on Windows
+///
+
+// REQUIRES: windows
+
+/// ###########################################################################
+/// test that no bfloat16 libraries are added in JIT mode
+// RUN: %clangxx -fsycl %s --sysroot=%S/Inputs/SYCL -### 2>&1 \
+// RUN:   | FileCheck %s -check-prefix=BFLOAT16
+
+// test that no bfloat16 libraries are added in JIT mode with generic target
+// RUN: %clangxx -fsycl -fsycl-targets=spir64 %S/Inputs/SYCL/c.cpp -### 2>&1 \
```

Comment:
```suggestion
// RUN: %clangxx -fsycl -fsycl-targets=spir64 %s -### 2>&1 \
```
Why add a separate file when you can just use `%s`?

Reply:
Yes, that will work since we are doing -###. Will remove that file.